tests/src/Interop/ICustomMarshaler/MultipleALCs/CustomLoadContext.cs

```diff
@@ -0,0 +1,48 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Runtime.Loader;
+
+class CustomLoadContext : AssemblyLoadContext
+{
+    private string unmanagedDirectory;
+
+    internal CustomLoadContext(string unmanagedDirectory)
+    {
+        this.unmanagedDirectory = unmanagedDirectory;
+    }
+
+    protected override Assembly Load(AssemblyName assemblyName)
```

Comment:
Looks to me that this function is not necessary for the test and could be implemented as

```C#
protected override Assembly Load(AssemblyName assemblyName) { return null; }
```